### PR TITLE
metrics: add ddl execution duration metrics

### LINKED ISSUE #2595: add prometheus metrics for sink execution ddl
## Feature Request

**Is your feature request related to a problem? Please describe:**

Add Prometheus metrics for monitoring time cost of sink execute DDL.

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### LINKED ISSUE #2669: CDC clould: grafana -> TiCDC ->Changefeed -> Changefeed checkpoint lag, display issue
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

- Grafana -> TiCDC, last 3 hours.

2. What did you expect to see?

3. What did you see instead?

- TiCDC ->Changefeed -> Changefeed checkpoint lag 
image

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
        4.0.14
        ```

    - TiCDC version (execute `cdc version`):

        ```
        # ./cdc version
Release Version: 4.0.14
Git Commit Hash: bbfda297e0e5ddbfb5e3386938be50758c74bdbb
Git Branch: cloud-cdc-4.0
UTC Build Time: 2021-08-26 14:08:37
Go Version: go version go1.16.3 linux/amd64
Failpoint Build: false

        ```

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve? 
try to close #2595, also fix #2669

### What is changed and how it works?

* add histogram as metric for DDL execution duration.
* add grafana panel to monitor it
  * MySQL sink ddl execution duration -> Heatmap
  * MySQL sink ddl execution duration percentile -> graph
* update some other old grafana panel.
  * Processor Memory Consumption Per Changefeed.   table -> graph 
  * changefeed checkpoint lag -> set min to 0 for yAxis

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

Side effects

Related changes

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: cdc/sink/metrics.go [hunk 1]
```diff
@@ -34,6 +34,14 @@ var (
 			Help:      "Bucketed histogram of processing time (s) of a txn.",
 			Buckets:   prometheus.ExponentialBuckets(0.002 /* 2 ms */, 2, 18),
 		}, []string{"capture", "changefeed"})
+	execDDLHistogram = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "ticdc",
+			Subsystem: "sink",
+			Name:      "ddl_exec_duration",
+			Help:      "Bucketed histogram of processing time (s) of a ddl.",
+			Buckets:   prometheus.ExponentialBuckets(0.002, 2, 18),
```
Comment (overvenus) [inline cdc/sink/metrics.go:43]:
```suggestion
			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 18),
```

For DDL events, we cares long running events, so I think we can range from 10ms to 43min

**Review by overvenus — COMMENTED** [review]
Rest LGTM

**Review by overvenus — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Comment by amyangfei** [comment]
```
# Time: 2021-08-31T10:30:31.704994657+08:00
# Txn_start_ts: 427393540858839044
# User@Host: root[root] @ 127.0.0.1 [127.0.0.1]
# Conn_ID: 37
# Query_time: 60.015354254
# Parse_time: 0.000059793
# Compile_time: 0.000251025
# Rewrite_time: 0.000177662
# Optimize_time: 0.000023462
# Wait_TS: 0.000008984
# Cop_time: 0.022757177 Process_time: 0.008 Wait_time: 0.013 Backoff_time: 0.036 Request_count: 109 Total_keys: 109 Rocksdb_block_cache_hit_count: 110
# Is_internal: false
# Digest: 35f974b0c4b120ba15debc74208369d8d0f2432cb2ca681f53b0e7fb66b327a0
# Stats: t1:pseudo
# Num_cop_tasks: 109
# Cop_proc_avg: 7.3394e-05 Cop_proc_p90: 0 Cop_proc_max: 0.001 Cop_proc_addr: 127.0.0.1:20160
# Cop_wait_avg: 0.000119266 Cop_wait_p90: 0.001 Cop_wait_max: 0.001 Cop_wait_addr: 127.0.0.1:20160
# Cop_backoff_regionMiss_total_times: 18 Cop_backoff_regionMiss_total_time: 0.036 Cop_backoff_regionMiss_max_time: 0.002 Cop_backoff_regionMiss_max_addr: 127.0.0.1:20160 Cop_backoff_regionMiss_avg_time: 0.002 Cop_backoff_regionMiss_p90_time: 0.002
# Mem_max: 394
# Prepared: false
# Plan_from_cache: false
# Plan_from_binding: false
# Has_more_results: false
# KV_total: 899.989907111
# PD_total: 0.015390817
# Backoff_total: 0.056
# Write_sql_response_total: 0
# Succ: false
# Plan: tidb_decode_plan('/gSoMAkyOV8zCTAJMAlOL0EJMAl0aW1lOjFtMHMsIGxvb3BzOjEJMCBCeXRlcwEhkAoxCTMxXzYJMAkxMDAwMAlkYXRhOlRhYmxlRnVsbFNjYW5fNQlORQDwPiwgY29wX3Rhc2s6IHtudW06IDEwOSwgbWF4OiA3MDYuN8K1cywgbWluOiAyODcuMsK1cywgYXZnOiA0MjIuNgUeJHA5NTogNjAwLjgFDzR0b3RfcHJvYzogOG1zLAUPHHdhaXQ6IDEzARAMcnBjXwlpCDM3LAUOBdsQIDU1LjcBIPBPY29wcl9jYWNoZV9oaXRfcmF0aW86IDAuMDB9LCBiYWNrb2Zme3JlZ2lvbk1pc3M6IDM2bXN9CTM5NCBCeXRlcwlOL0EKMgk0M181CTFfMAkpEaB0YWJsZTp0MSwga2VlcCBvcmRlcjpmYWxzZSwgc3RhdHM6cHNldWRvCSElCGt2XyUSEHtwcm9jJQwEMW0tBiGACHA4MAkIBDk1CRkUaXRlcnM6JTkBOAULVH0sIHNjYW5fZGV0YWlsOiB7dG90YWwlGShlc3Nfa2V5czogMCUfBGFsDQ8FQnRyb2Nrc2RiOiB7ZGVsZXRlX3NraXBwZWRfY291bnQFNAhrZXlKFgAMYmxvYyHGAGM1PA0yHDExMCwgcmVhLkMABQ8YYnl0ZTogMElHLH19fQlOL0EJTi9BCg==')
# Plan_digest: 70e316e5e20fbf2e292b74bfffe36219c403212e6ccd33f9c9c578f415e06743
delete from region_merge.t1;
```

**Pull request merged**